### ISSUE TITLE
Making the tray icon function more practical

### ISSUE THREAD

**Issue by leafcutterant** [standardnotes/desktop#422]
First, I'd like to say massive thanks for implementing the tray icon as described in #103! This made SN much more usable as an app that constantly accompanies daily tasks.

I'd like to propose two modifications to the tray icon behavior. These make the behavior more practical, but it may also be more intuitive to users who have been users of tray icons in other apps.

1. **Make the left clicking on the tray icon display the app, and leave the "Hide/Show/Quit" menu to the right click.** I'd say this is quite standard among tray icon implementations. Having to go through a menu to display the app is an unnecessary extra step.
2. **Don't display a tray icon when SN is displayed in the task bar.** There is no rough consensus on this (e.g. Thunderbird works like this, but Zim doesn't). But the argument for this is that an aim of tray iconization is to save space on the task bar. By the same logic, when the app is *not* minimized to the tray and is visible in the task bar, there's little use of displaying a tray icon as well, and it takes up extra space (since the task bar and the tray icon area are basically vying for the same space).

**Comment by mobitar** [comment]
Indeed, I would like to make right clicking open the app immediately.

**Comment by leafcutterant** [comment]
> I would like to make right clicking open the app immediately.

You mean left clicking, right?

**Comment by mobitar** [comment]
Right, sorry.

**Comment by marcopiraccini** [comment]
It would be good to have also a startup param to run directly in tray (like `--tray`). The reason is that I would like to have it running at startup "silently" and park in the tray. 
Should be easy, if you agree I can create a PR for this.

**Comment by mobitar** [comment]
Where would the user run this param?

**Comment by marcopiraccini** [comment]
When setting for autostart. The use case is: I want standardnotes to start automatically and go in the trail (like slack) without opening the main window.

**Comment by baptiste-grob** [comment]
Hey @leafcutterant, I've just created a PR that make the tray icon more aligned with what other Windows apps are doing.
As far as removing the tray icon on focus, after some consideration and experimentation I've found that removing the tray icon makes windows forget its position. That is, if you had previously dragged the icon to a specific place in the tray, that position would be lost the next time the icon would appear. So I don't believe breaking system behavior is a good idea here.
